config/cron_udp.hjson

```diff
@@ -41,8 +41,8 @@
         left join entity.course_grade cg
```

Comment:
The Cron passing params changed with 2 library upgrade ( SQLAlchemy and psycopg). I felt some point we should transition our Postgres Queries to BigQuery for simplicity.

Reply:
True! I've created #1557 to track this request.